docs/compilation.rst

```diff
@@ -422,14 +424,14 @@ Following CMake parameters could be helpful in setting the right paths:
 * protobuf options can be obtained from |protobuf_cmake link|_
 
 
-Cross-compiling for aarch64 Linux (Docker)
-------------------------------------------
+Cross-compiling for aarch64 Jetson Linux (Docker)
+-------------------------------------------------
 
 .. note::
 
   Support for aarch64 Linux platform is experimental. Some of the features are available only for
-  x86-64 target and they are turned off in this build. There is no support for DALI Python library
-  on aarch64 yet. Some Operators may not work as intended due to x86-64 specific implementations.
+  x86-64 target and they are turned off in this build. Some Operators may not work as intended due
```

Comment:
```suggestion
  x86-64 target and they are turned off in this build. Some operators may not work as intended due
```

Reply:
I have mixed feelings about this sentence. Why should we even say such things?

Reply:
I'd also vote for removing this

Reply:
Done